alacritty_terminal/src/clipboard.rs

```diff
@@ -1,98 +1 @@
-#[cfg(not(any(target_os = "macos", target_os = "windows")))]
-use std::ffi::c_void;
 
```

Comment:
While the efficiency of this code is outstanding, I think it might be better to just remove this file.